### PR TITLE
[DOC] update reference link for unece code

### PR DESCRIPTION
**Pull request by joanna350**
Description of the issue/feature this PR addresses: outdated reference doc (under addons/account_edi for v15.0)

Current behavior before PR: redundant googling

Desired behavior after PR is merged: refer to the correct ref for further correction as fits

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/cla/individual/joanna.md
Comment (mart-e) [inline doc/cla/individual/joanna.md:1]: sorry but "Digital Nomad" is not a country

**Review by vin-odoo — APPROVED** [review]
Sure, why not :smile:

**Comment by joanna350** [comment]
@vin-odoo thanks for approving! feel free to merge when the time comes

**Comment by vin-odoo** [comment]
@joanna350 You should correct the CLA according to the [doc](https://github.com/odoo/odoo/blob/15.0/doc/cla/sign-cla.md), see mart-e comment.
Also, not sure why you closed the PR if you still expect it to be merged :man_shrugging:

**Comment by joanna350** [comment]
I’ve noted the request to edit the CLA. It just seemed yet to be necessary for a non-critical, stable contribution

**Comment by mart-e** [comment]
Thanks for the update. I have squashed the commits, looks good now :+1: 
@robodoo r+

**Comment by joanna350** [comment]
I see why my local branch diverged from remote, thanks for minding. You can however leave it with me next time :)

**Comment by mart-e** [comment]
As we have some specific [git guidelines](https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#git) on commit messages or formating that contributors may not be aware of (or unsure how to apply/amend), we can do it for them. This can also be seen as an intrusion, sorry about that :)
When you create a PR, you should see and option "allow edit from maintainers" ([cf doc](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)) that you can disable if you wish.

**Comment by joanna350** [comment]
Ofc, thank you for sharing. I agree it is cleaner this way.

**Pull request closed without merging**
